### PR TITLE
Added Email action

### PR DESCRIPTION
**Pull request by nishu88**
#59 Added a file called sendemailaction.py to actions directory and modified the action package accordingly. Used smtplib module in python3 :)

### CODE REVIEW

File: pastepwn/actions/sendemailaction.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        super().__init__()
+
+        if not re.match(r"^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$", username) or username is None:
+            raise ValueError("username not correct or None!")
+        if not re.match(r"^\w+([\.-]?\w+)*@\w+([\.-]?\w+)*(\.\w{2,3})+$", reciever_email) or reciever_email is None:
+            raise ValueError("reciever_email not correct or None!")
+        self.username = username
+        self.password=password
+        self.reciever_email=reciever_email
+        self.hostname="smtp."+hostname+".com"
```
Comment (d-Rickyy-b) [inline pastepwn/actions/sendemailaction.py]: This prevents domains apart from `.com` domains. Better would be to have the user insert the exact smtp domain themself => `self.hostname = hostname`

File: pastepwn/actions/sendemailaction.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        self.port=port
+
+
+    def perform(self, paste, analyzer_name=None):
+        address = self.username
+        password =self.password
```
Comment (d-Rickyy-b) [inline pastepwn/actions/sendemailaction.py]: Missing space around `=`

File: pastepwn/actions/sendemailaction.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+        mailto = self.reciever_email
+        msg = 'Subject: {}\n\n{}'.format("From Pastepwn", str(paste))
+        mailServer = smtplib.SMTP(self.hostname , self.port)
+        mailServer.starttls()
+        mailServer.login(address ,password)
+        mailServer.sendmail(address, mailto , msg)
+        print(" \n Sent!")
```
Comment (d-Rickyy-b) [inline pastepwn/actions/sendemailaction.py]: Please don't use print. I know there is currently no logging, but prints will be left out in the log file. Best would be to simply remove this line.

**Comment by nishu88** [comment]
@d-Rickyy-b Thank you for your time in reviewing my request. I've made changes according to your suggestions. Thank you for being as descriptive as possible with the review ;)
Cheers :)

**Comment by d-Rickyy-b** [comment]
Sadly you deleted your branch, so I cannot merge this PR anymore. Thanks for your participation though!

**Pull request closed without merging**